app/javascript/app/components/emission-pathways-scenario-table/emission-pathways-scenario-table-selectors.js

```diff
@@ -12,13 +17,60 @@ const getIdData = createSelector([getData, getId], (data, id) => {
   return data.find(d => String(d.id) === id) || null;
 });
 
-const getIndicatorsData = createSelector(getIdData, data => {
+const getIndicatorIds = createSelector(getIdData, data => {
   if (!data) return null;
-  return data.indicators || null;
+  return data.indicators.map(i => i.id) || null;
 });
 
-export const filteredDataBySearch = createSelector(
-  [getIndicatorsData, getQuery],
+const scenarioIndicatorsData = createSelector(
```

Comment:
I dont see what this does? Surely just get the data in the getCategories selector and map to the options array?